### PR TITLE
[AttendeeArea] updating to display all the display names

### PR DESCRIPTION
**Pull request by JoshuaLai**
Quick PR to list out the participants

Screen Shot 2021-06-09 at 4 43 12 PM

### CODE REVIEW

File: Chat/ClientApp/src/components/Room.tsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ export default (props: RoomProps): JSX.Element => {
     <Stack className={chatScreenContainerStyle}>
       <ChatHeader leaveChatHandler={() => { }} selectedPane={SidePanelTypes.None} setSelectedPane={() => { }} />
       <Stack className={chatScreenBottomContainerStyle} horizontal={true}>
-        <LeftPane />
+        <LeftPane onTabClicked = {() => {} }/>
```
Comment (minnieliu) [inline Chat/ClientApp/src/components/Room.tsx]: I will be getting rid of this file :)

File: Chat/ClientApp/src/containers/AttendeesArea.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { connect } from 'react-redux';
+import AttendeesArea from '../components/AttendeesArea';
+import { State } from '../core/reducers';
+
+const mapStateToProps = (state: State) => ({
+  eventAttendees: state.threadMembers.threadMembers
```
Comment (minnieliu) [inline Chat/ClientApp/src/containers/AttendeesArea.ts:6]: Why do you need .threadMembers.threadMembers here?

Comment (JoshuaLai) [reply Chat/ClientApp/src/containers/AttendeesArea.ts:6]: I agree its an odd name but it is just the chain of properties to get all the members in the chat

File: Chat/ClientApp/src/components/Room.tsx (hunk 1 above)
Comment (JoshuaLai) [reply Chat/ClientApp/src/components/Room.tsx]: Yep this will be removed since your PR has been merged

**Review by minnieliu — APPROVED** [review]

**Pull request merged**